byteparse/jvm/src/test/scala/byteparse/LargeBmpIteratorTests.scala

```diff
@@ -0,0 +1,60 @@
+package byteparse
+
+import java.net.URL
+
+import fastparse.{IteratorParserInput, LoggedParsedInput}
+
+import utest._
+import fastparse.allByte._
+import BmpParser._
+
+object LargeBmpIteratorTests extends TestSuite {
+  val url = "https://raw.githubusercontent.com/lihaoyi/fastparse/master/byteparse/jvm/src/test/resources/lena.bmp"
+  def stream = new Iterator[IndexedSeq[Byte]] {
```

Comment:
Could we make `parseIterator` take an `Iterator[Array[Byte]]`? I'm pretty sure `IndexedSeq[Byte]` ends up take 5-20x as much memory as `Array[Byte]` due to boxing converting each byte to a 4-8 byte pointer to an object with a 8-12 byte header with 1 byte of data each...